### PR TITLE
Update azure-blobs and fix API hash

### PR DESCRIPTION
**Pull request by rouming**
PR does the following:

1. returns beloved 'make proto-vendor'
2. updates azure-blobs to the latest version (request from other users of the lib)
3. fixes /api hash

cc: @deitch

### CODE REVIEW

**Comment by christoph-zededa** [comment]
LGTM

**Review by deitch — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -803,6 +803,9 @@ cache-export-docker-load: $(LINUXKIT)
 ## will skip image if not found in cache
 cache-export-docker-load-all: $(LINUXKIT) $(addsuffix -cache-export-docker-load,$(PKGS_DOCKER_LOAD))
 
+proto-vendor:
+	@$(DOCKER_GO) "cd pkg/pillar ; go mod vendor" $(CURDIR) proto
+
```
Comment (eriknordmark) [inline Makefile:808]: Should we document this? And why don't we also the libs-vendor and all-vendor targets?

**Review by eriknordmark — APPROVED** [review]
Kick off tests (to test new azure-storage-blob version works)

File: Makefile (hunk 1 above)
Comment (rouming) [reply Makefile:808]: I do not know. We got used to this command and find it very convenient, so I just bring it back to live :)

Comment (rouming) [reply Makefile:808]: It's actually documented in the make help. Seems enough

**Pull request merged**